src/components/JobStatusList.tsx

```diff
@@ -79,7 +79,8 @@ export class JobStatusList extends React.Component<Props, void> {
   }
 
   private scrollToSelectedJob() {
-    const row = document.querySelector(`.${jobStatusStyles.isActive}`)
+    const job = this.props.selectedFeature as beachfront.Job
```

Comment:
I'm getting this exception on job deselect. I think you're just missing a check to make sure `selectedJob` isn't null or undefined.
```
Uncaught TypeError: Cannot read property 'properties' of null
    at JobStatusList.scrollToSelectedJob (JobStatusList.tsx?027f:83)
    at JobStatusList.componentDidUpdate (JobStatusList.tsx?027f:41)
    at ReactCompositeComponentWrapper.invokeComponentDidUpdateWithTimer (ReactCompositeComponent.js?195a:65)
    at CallbackQueue.notifyAll (CallbackQueue.js?f228:67)
    at ReactReconcileTransaction.close (ReactReconcileTransaction.js?51cf:81)
    at ReactReconcileTransaction.closeAll (Transaction.js?fbf2:204)
    at ReactReconcileTransaction.perform (Transaction.js?fbf2:151)
    at ReactUpdatesFlushTransaction.perform (Transaction.js?fbf2:138)
    at ReactUpdatesFlushTransaction.perform (ReactUpdates.js?371c:90)
    at Object.flushBatchedUpdates (ReactUpdates.js?371c:173)
```